crates/basilica-sdk-python/tests/test_rl_client.py

```
@@ -308,3 +308,69 @@ def test_delete_cluster_not_found_maps_per_the_error_table(server):
+    rec.responses = [(200, {"name": "p", "uid": "u", "phase": "Provisioning"})]
+    rl(base).create_cluster(base_model="m/m", gpu_model="H100")
+    (r,) = rec.requests
+    assert "relay" not in r["body"]
+
+
+def test_rotate_credentials_wire_shape(server):
+    base, rec = server
+    rec.responses = [(200, {"name": "byo-pool",
+                            "rotatedAt": "2026-08-31T12:00:00+00:00"})]
+    out = rl(base).rotate_credentials(
+        "byo-pool", access_key_id="NEWAK", secret_access_key="NEWSK"
```

Ruff (0.16.2)

[error] Possible hardcoded password assigned to argument: "secret_access_key"

(S106)